hugegraph-llm/src/hugegraph_llm/operators/index_op/semantic_id_query.py

```diff
@@ -17,28 +17,38 @@
 
 
 import os
-from typing import Dict, Any, Literal
+from copy import deepcopy
+from typing import Dict, Any, Literal, List, Tuple
 
+from pyhugegraph.client import PyHugeClient
 from hugegraph_llm.config import resource_path, settings
 from hugegraph_llm.indices.vector_index import VectorIndex
 from hugegraph_llm.models.embeddings.base import BaseEmbedding
 
 
 class SemanticIdQuery:
+    ID_QUERY_TEMPL = "g.V().hasId({vids_str})"
```

Comment:
```suggestion
    ID_QUERY_TEMPL = "g.V({vids_str})"
```

no need to use extra `hasId/hasLabel` step